### DISCUSSION TITLE
manageDeclarationPartition: bevallás partíció maximum 128 MB

### DISCUSSION THREAD

**Discussion by szalaierika** [nav-gov-hu/eVAT#48] [Kérdések és válaszok - Gépi interfész] [answered]
Kedves NAV Szakértők!

A dokumentáció alapján:

"A fájlfeltöltés során a manageDeclarationPartition operáció esetén a feltölteni kívánt bevallás partíció maximum 128 MB, a manageAttachmentUpload operáció esetén a feltölteni kívánt melléklet 100 MB lehet a kérésben szereplő octet-stream mérete."

Azt szeretnénk megkérdezni, hogy hány adatrekorddal kalkuláltak az 1* 128 MB méretkorlát meghatározásakor?

A választ előre is köszönjük!

**Comment by renced42** [comment, marked as answer]
Pontosan nem tudom de kb 50.000 rekorddal.
